SimPEG/Examples/PF_Gravity_Laguna_del_Maule_Inversion.py

```diff
@@ -0,0 +1,247 @@
+# coding: utf-8
+'''
```

Comment:
Can this be moved to document the run function (that way we pick it up in the docs). 

After, run `python __init__.py` again!

Can you also please use `"""` doc quotes rather than `'''`. :)